### PR TITLE
Provide early non-nil status

### PR DESCRIPTION
**Pull request by edwardcallahan**
Sets `Status.LastUpdate` when status is nil. 
Hand pick and -certified #28

### CODE REVIEW

**Comment by marcoderama** [comment]
Looks like you have to rebase this Ed.

**Comment by edwardcallahan** [comment]
Yes. I missed #24, thanks.

**Comment by edwardcallahan** [comment]
This passed certification!

File: deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Application Runtime
     certified: 'false'
-    containerImage: 'lightbend-docker-testing-registry.bintray.io/akka-cluster-operator-certified:0.0.4'
+    containerImage: 'registry.connect.redhat.com/lightbend/akka-cluster-operator:latest'
```
Comment (marcoderama) [inline deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml:10]: Do you want that to be 
registry.connect.redhat.com/lightbend/akka-cluster-operator **-certified** :latest

File: deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -157,7 +157,7 @@ spec:
                             fieldPath: metadata.name
                       - name: OPERATOR_NAME
                         value: akka-cluster-operator
-                    image: lightbend-docker-testing-registry.bintray.io/akka-cluster-operator-certified:0.0.4
+                    image: registry.connect.redhat.com/lightbend/akka-cluster-operator:latest
```
Comment (marcoderama) [inline deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml:160]: ditto?

File: .github/workflows/push.yml [hunk 3]
```diff
@@ -12,17 +12,15 @@ jobs:
     - name: operator-sdk
       uses: ./.github/action/operator-sdk
       with:
-        args: build lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator:latest
+        args: build lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator-certified:latest
     - name: bintray login
-      uses: actions/docker/login@8cdf801b322af5f369e00d85e9cf3a7122f49108
+      run: docker login
```
Comment (marcoderama) [inline .github/workflows/push.yml:17]: I see you trimmed down the login command...  Was that intentional or is a rebase in order?

File: deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml (hunk 1 above)
Comment (edwardcallahan) [reply deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml:10]: Good question. 
No. Here's why:
image

File: deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml (hunk 2 above)
Comment (edwardcallahan) [reply deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml:160]: ditto

File: .github/workflows/push.yml (hunk 3 above)
Comment (edwardcallahan) [reply .github/workflows/push.yml:17]: This is a PR into `certified`. This was intentional. The rebase wasn't worth the effort here.

File: deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml (hunk 1 above)
Comment (marcoderama) [reply deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml:10]: I see.  So the
> `docker push lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator-certified:latest`

in `push.yml` is just so we have our own copy?

Comment (edwardcallahan) [reply deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml:10]: Good question. I've setup the build config in the certified project setup. Maybe better to just remove this publish target.

Comment (edwardcallahan) [reply deploy/olm-catalog/akka-cluster-operator/0.0.1/akka-cluster-operator-certified.v0.0.1.clusterserviceversion.yaml:10]: Removed push.yml.

**Comment by marcoderama** [comment]
Rather than removing push.yml, I think you could have qualified the branch(es) that it applied to.

**Review by marcoderama — COMMENTED** [review]
Yeah, I think that's better.
(But I have to mention that you're updating an old version of `push.yml` cuz no rebase from master yet...  This version doesn't even work.)

**Review by marcoderama — APPROVED** [review]
Approved.

**Pull request merged**